src/components/Link/Link.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from '../../utils/cx';
+import './link.css';
+
+export const linkModifiers = [
+    'primary',
+    'muted',
+    'text'
+];
+
+const Link = ({ modifier = 'primary', disabled = false, children, className = '', onClick = () => {}, linkTag = '#', ...rest }) => {
```

Comment:
@Kathuria please change ```listTag``` prop to ```href```

Reply:
Ok, Fixed - will be available in next commit